src/NHSD.BuyingCatalogue.Data/Infrastructure/DbConnectionFactory.cs

```diff
@@ -17,8 +17,7 @@ internal sealed class DbConnectionFactory : IDbConnectionFactory
         /// <summary>
         /// Initialises a new instance of the <see cref="DbConnectionFactory"/> class.
         /// </summary>
-        public DbConnectionFactory(ISettings settings)
-            => _settings = settings.ThrowIfNull(nameof(settings));
+        public DbConnectionFactory(ISettings settings) => _settings = settings ?? throw new ArgumentNullException(nameof(settings));
```

Comment:
I'm perfectly content with doing it this way, however, I need to ask whether it would be good (for consistency sake) to either do it this way everywhere where applicable ( changes to [src/NHSD.BuyingCatalogue.API/Infrastructure/Settings.cs](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueService/pull/183/files#diff-9e9b23fe77236c198e923bbd968f823a) would be a good example)
or change it, so it is done the same way as in the example given. 

What do you think?

Reply:
I've changed the Settings class to be written as an expression body too